lib/models/server-watcher.js

```diff
@@ -14,6 +14,10 @@ module.exports = class ServerWatcher extends Watcher {
     return new (require('sane'))(this.watchedDir, options);
   }
 
+  constructBroccoliWatcher(options) {
+    return new (require('sane'))(this.watchedDir, options);
```

Comment:
This is exactly the same as `constructWatcher`, what is the point of having a second identical method?

Reply:
This has to do with some complexity with how the inheritance + feature flagging. 
It likely could be refactored + given a comment. 

Once the flag is gone, this wont be as silly.

Reply:
I cleaned this up a bit as well but stef's point above is the underlining cause for both of these nearly identical methods.